### ISSUE TITLE
Future enhancement

### ISSUE THREAD

**Issue by TheElectronicMusicProject** [andrea-cpu96/Smart-Watch#1]
FYI: [CrowPanel -ESP32 Display-1.28(R) inch 240*240 Round IPS Display Capacitive Touch SPI Screen](https://www.elecrow.com/crowpanel-esp32-display-1-28-r-inch-240-240-round-ips-display-capacitive-touch-spi-screen.html)

This could help to improve the project with more additional features.

**Comment by andrea-cpu96** [comment]
> FYI: [CrowPanel -ESP32 Display-1.28(R) inch 240*240 Round IPS Display Capacitive Touch SPI Screen](https://www.elecrow.com/crowpanel-esp32-display-1-28-r-inch-240-240-round-ips-display-capacitive-touch-spi-screen.html)
> 
> This could help to improve the project with more additional features.

Interesting, but the goal of this project is that of creating a digital watch from scratch, with custom hardware and firmware.

**Comment by TheElectronicMusicProject** [comment]
Yes, I know. I thought it can be useful to find inspiration for more features
